libs/server/Objects/List/ListObject.cs

```diff
@@ -57,6 +57,7 @@ public enum OperationDirection
     /// </summary>
     public partial class ListObject : GarnetObjectBase
     {
+        /// <inheritdoc />
         readonly LinkedList<byte[]> list;
```

Comment:
Inheritdoc from where? Same question for SetObject

Reply:
Sorry, this was related to something I did before & forgot to remove, deleting these...